src/go/intelligence/pipeline/fetch_alert.go

```diff
@@ -14,17 +14,20 @@ import (
 //FetchAlert - function for fetching all active alerts from AlertManager
 func FetchAlert() <-chan models.AmJSON {
 	fetchedData := make(chan models.AmJSON)
+
```

Comment:
There's one specific reason for pulling this block out of goRoutine. I want to first fetch all alerts and then start the concurrent calls. This is because there was a discrepancy with matching with Alerts going to be processed and already silenced alerts. So to make sure we all have alert data before calling goroutines.